### PR TITLE
atlantis: 0.16.1 -> 0.19.2

### PR DESCRIPTION
**Pull request by bryanasdev000**
###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/atlantis/default.nix [hunk 1]
```diff
@@ -2,21 +2,24 @@
...
   };
 
-  vendorSha256 = null;
-
-  doCheck = false;
+  vendorSha256 = "sha256-ux+Hw/TjeiY9VYhIQxaltZGk5CkxAab8R7kAsTaMUGc=";
 
   subPackages = [ "." ];
 
+  doInstallCheck = true;
```
Comment (jpotier) [inline pkgs/applications/networking/cluster/atlantis/default.nix:18]: That's a really cool feature I wasn't aware of :+1:

**Comment by jpotier** [comment]
@bryanasdev000 is this ready to be reviewed? Currently, it is marked as Draft.

**Comment by bryanasdev000** [comment]
> @bryanasdev000 is this ready to be reviewed? Currently, it is marked as Draft.

It is, but I didn't have chance to test much.

I will change to ready to review.

**Review by jpotier — APPROVED** [review]
```
 % nix run https://github.com/bryanasdev000/nixpkgs/archive/refs/heads/atlantis0192.tar.gz#atlantis -- --help                                                                    /tmp wrk
Terraform Pull Request Automation

Usage:
  atlantis [command]

Available Commands:
  help        Help about any command
  server      Start the atlantis server
  testdrive   Start a guided tour of Atlantis
  version     Print the current Atlantis version

Flags:
  -h, --help   help for atlantis

Use "atlantis [command] --help" for more information about a command.
```

built and tried running the executable, with success.

0.19.2 is indeed the latest release available upstream.

Inspecting the build log, one can indeed see that the install check ran successfully:

```
% nix log /nix/store/4bpgx1m8qzrx65mlkzc0w7zx6whb4qsq-atlantis-0.19.2                                                                                                           /tmp wrk
...
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/4bpgx1m8qzrx65mlkzc0w7zx6whb4qsq-atlantis-0.19.2
shrinking /nix/store/4bpgx1m8qzrx65mlkzc0w7zx6whb4qsq-atlantis-0.19.2/bin/atlantis
strip is /nix/store/j0lglxsmvrd3zyv9dlabffc0n0xnxbia-gcc-wrapper-10.3.0/bin/strip
stripping (with command strip and flags -S) in /nix/store/4bpgx1m8qzrx65mlkzc0w7zx6whb4qsq-atlantis-0.19.2/bin 
patching script interpreter paths in /nix/store/4bpgx1m8qzrx65mlkzc0w7zx6whb4qsq-atlantis-0.19.2
checking for references to /tmp/nix-build-atlantis-0.19.2.drv-0/ in /nix/store/4bpgx1m8qzrx65mlkzc0w7zx6whb4qsq-atlantis-0.19.2...
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
(END)
```

**Review by thiagokokada — APPROVED** [review]

**Pull request merged**